distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/NsiliClient.java

```diff
@@ -755,7 +748,7 @@ public String getIorTextFile(String iorURL) throws Exception {
         }
```

Comment:
Added a try/catch and LOGGER.error() that will report errors when the client cannot connect